monai/utils/module.py

```diff
@@ -564,11 +567,11 @@ def version_leq(lhs: str, rhs: str) -> bool:
     """
 
     lhs, rhs = str(lhs), str(rhs)
-    pkging, has_ver = optional_import("pkg_resources", name="packaging")
+    pkging, has_ver = optional_import("packaging", name="packaging")
```

Comment:
pkging is even not used here.
```suggestion
    pkging, has_ver = optional_import("packaging")
```

Reply:
Hello,
We removed references to pkging and pkg_resources but optional_import returns a tuple. Do you want us to keep pkging in the optional_import or is there any other suggestions you had?